app/src/main/java/com/nextcloud/talk/adapters/items/AppItem.java

```diff
@@ -25,26 +25,25 @@
 import android.text.SpannableString;
 import android.text.style.ForegroundColorSpan;
 import android.view.View;
-import android.widget.ImageView;
-import android.widget.TextView;
-import androidx.annotation.Nullable;
-import butterknife.BindView;
-import butterknife.ButterKnife;
+
 import com.nextcloud.talk.R;
 import com.nextcloud.talk.application.NextcloudTalkApplication;
+import com.nextcloud.talk.databinding.RvItemAppBinding;
+
+import java.util.List;
+
+import androidx.annotation.Nullable;
 import eu.davidea.flexibleadapter.FlexibleAdapter;
 import eu.davidea.flexibleadapter.items.AbstractFlexibleItem;
 import eu.davidea.flexibleadapter.items.IFlexible;
 import eu.davidea.viewholders.FlexibleViewHolder;
 
-import java.util.List;
-
 public class AppItem extends AbstractFlexibleItem<AppItem.AppItemViewHolder> {
```

Comment:
AppItem is not used anywehere...?

Reply:
this one should have been deleted with
https://github.com/nextcloud/talk-android/pull/1285/files
as i simplified the "share to" feature..